public/components/health-check/services/check-index-pattern/check-index-pattern.service.ts

```diff
@@ -29,13 +46,34 @@ const checkPattern = async (appConfig, checkLogger: CheckLogger) =>  {
 
 const decoratorHealthCheckRunCheckEnabled = (checkKey, fn) => {
   return async (appConfig: any, checkLogger: CheckLogger) => {
-    if(appConfig.data[`checks.${checkKey}`]){
+    if (appConfig.data[`checks.${checkKey}`]) {
       await fn(appConfig, checkLogger);
-    }else{
+    } else {
       checkLogger.info(`Check [${checkKey}]: disabled. Skipped.`);
     };
   }
 };
 
 const checkTemplate = decoratorHealthCheckRunCheckEnabled('template', checkTemplateService);
 const checkFields = decoratorHealthCheckRunCheckEnabled('fields', checkFieldsService);
+
+const checkMaxBuckets = decoratorHealthCheckRunCheckEnabled('maxBuckets',
+  (appConfig, checkLogger) => checkPluginPlatformSettings(
```

Comment:
**suggestion**: The `checkPluginPlatformSettings` could be refactored to accept the `appConfig` and `checkLogger` parameters instead of currying form.

I checked the `checkPluginPlatformSettings` is not used in other places.

Reply:
Thanks for the suggestion!